### ISSUE TITLE
Webhooks/Zapier Integration

### ISSUE THREAD

**Issue by michaelpanik** [bordosocial/bordo#81]
Either webhooks or native Zapier integration to allow users to integrate with anything in the Zapier ecosystem.

**Comment by chevinbrown** [comment]
I looked some into zappier yesterday. That may be a larger integration than I originally thought, but would be super cool.

**Comment by michaelpanik** [comment]
Yeah, it might be a lot. The good news is that Zapier can take any webhook as a trigger, so if we can even implement webhooks, we can have it as a phase one.
